Files/Files.csproj

```diff
@@ -986,28 +986,28 @@
       <Version>7.0.0-preview4</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp">
-      <Version>6.1.1</Version>
+      <Version>7.0.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.DeveloperTools">
-      <Version>6.1.1</Version>
+      <Version>7.0.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.Notifications">
-      <Version>6.1.1</Version>
+      <Version>7.0.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.UI">
```

Comment:
You used to get the Behaviors included with this package, now you don't so you need to add the Behaviors package explicitly to your dependencies. https://www.nuget.org/packages/Microsoft.Xaml.Behaviors.Uwp.Managed/

Reply:
Thank you! I didn't realize that there used to be another dependency.